internal/resources/resource_volume.go

```diff
@@ -256,11 +262,16 @@ func resourceMetalVolumeRead(d *schema.ResourceData, meta interface{}) (err erro
 	loc, _ := p.GetLocationName(volume.LocationID)
 	d.Set(vLocation, loc)
 	d.Set(vLocationID, volume.LocationID)
+
 	if err = d.Set(vShareable, volume.Shareable); err != nil {
 		return err
 	}
+
 	d.Set(vState, volume.State)
 	d.Set(vStatus, volume.Status)
+	if err = d.Set(vWWN, volume.WWN); err != nil {
+		return fmt.Errorf("set WWN: %v", err)
+	}
```

Comment:
Needed an extra space between the previous non-assignment and this assignment.
```suggestion

	if err = d.Set(vWWN, volume.WWN); err != nil {
		return fmt.Errorf("set WWN: %v", err)
	}
```

Reply:
I'm wondering why we are inconsistently checking for error returned by d.Set? is this to catch possible incompatibility between api versions?

Reply:
> I'm wondering why we are inconsistently checking for error returned by d.Set? is this to catch possible incompatibility between api versions?

I think it is "historic": in new code, linter will flag calls done without check for error response. It was fine without checks a while back...